src/libraries/System.Private.CoreLib/src/System/Diagnostics/Tracing/RuntimeEventSource.cs

```diff
@@ -10,35 +10,30 @@ namespace System.Diagnostics.Tracing
     /// RuntimeEventSource is an EventSource that represents events emitted by the managed runtime.
     /// </summary>
     [EventSource(Guid = "49592C0F-5A05-516D-AA4B-A64E02026C89", Name = "System.Runtime")]
-    internal sealed class RuntimeEventSource : EventSource
+    internal sealed partial class RuntimeEventSource : EventSource
     {
+        internal const string EventSourceName = "System.Runtime";
+
         private static RuntimeEventSource? s_RuntimeEventSource;
         private PollingCounter? _gcHeapSizeCounter;
         private IncrementingPollingCounter? _gen0GCCounter;
         private IncrementingPollingCounter? _gen1GCCounter;
         private IncrementingPollingCounter? _gen2GCCounter;
-        private IncrementingPollingCounter? _exceptionCounter;
```

Comment:
Quick question on this - what was the criteria for moving some of these counters to RuntimeEventSource.CoreCLR.cs? Are the remaining counters something that Mono runtime supports?

Reply:
I kept things that compiled under Mono's S.P.C and moved things that currently don't have representation in Mono runtime. It is not the final and we might very well add back things once we continue to add support.

Reply:
Would it be better to just have MONO ifdefs around the ones that do not compile instead of moving them back and forth?

Reply:
I'm fine either way. Do we have any convention for things like this or is it handled case by case? Thought it made sense to split since support might not overlapping between runtimes, but agree that we might end up moving all back into the same file and then need to drop CoreCLR specific file at some point going forward.

Reply:
We do prefer split files for permanent factoring. For temporary splits or workarounds, ifdefs are ok.